IPIP/0328-gateway-json-cbor-response-format.md

```diff
@@ -1,4 +1,4 @@
-# IPIP 0328: Gateway JSON and TAR Response Formats
+# IPIP-328: JSON and CBOR Response Formats on HTTP Gateways
```

Comment:
@lidel all for it, but then we should update the template

Reply:
good call, done in 63ca6338d60505b17a9cb93b3b8b11c56a40d20a